src/h_vialib/client.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, secret, service_url=None, html_service_url=None):
         :param service_url: Location of the via server
         :param html_service_url: Location of the Via HTML presenter
         """
+        self._secure_secrets = SecureSecrets(secret.encode("utf-8"))
```

Comment:
No need to double guess inside `SecureSecrets`, here it's a `str`, conver it to `bytes`.

Reply:
I guess we could alternatively make it LMS and Via/Via HTML's jobs to pass in a byte string to h_vialib in the first place. They could either encode it or perhaps they could use [environb](https://docs.python.org/3/library/os.html#os.environb) to write it as a byte string directly from the environ. But this seems fine